development/orphan-remover/main.go

```diff
@@ -0,0 +1,331 @@
+// See https://cloud.google.com/docs/authentication/.
+// Use GOOGLE_APPLICATION_CREDENTIALS environment variable to specify
+// a service account key file to authenticate to the API.
+package main
```

Comment:
1. Add scripts that executes gofmt, golint, govet
2. Setup vendor with dep

Reply:
I am not sure what do you mean with the script, my local env is setup to run golint, guru goimports and gorename. 
Implemented vendoring with dep